docs/topics/forms/modelforms.txt

```diff
@@ -815,14 +815,43 @@ Model formsets
 
 Like :doc:`regular formsets </topics/forms/formsets>`, Django provides a couple
 of enhanced formset classes to make working with Django models more
-convenient. Let's reuse the ``Author`` model from above:
+convenient. To create a model formset out of the ``Author`` model from above,
+you have two options:
+
+Method 1: Using ``modelformset_factory``
+----------------------------------------
 
 .. code-block:: pycon
 
     >>> from django.forms import modelformset_factory
     >>> from myapp.models import Author
     >>> AuthorFormSet = modelformset_factory(Author, fields=["name", "title"])
 
+
+Method 2: Using ``ModelFormSet`` class
+--------------------------------------
+
+.. code-block:: pycon
+
+    >>> from django.forms import models
+    >>> from myapp.models import Author
+    >>> class AuthorFormSet(models.ModelFormSet):
+    ...     model = Author
+    ...     fields = ("name", "title")
+    ...
+
+Using the declarative syntax, you're able to define the formset directly as a
+class, similar to how you declare models.
+
+.. note::
+
+    When creating model formsets, you have the flexibility to customize their
```

Comment:
Do we need to adjust the wording below that says "This means that a model formset is an extension" (it only references the imperative syntax)?